### DISCUSSION TITLE
Can't write 2 operations on the Graphiql Editor and execute a selected one.

### DISCUSSION THREAD

**Discussion by saeta-eth** [apollographql/apollo-client-devtools#405] [General]
Hi @hwillson

I noticed that there is an issue trying to perform a query/mutation when users type more than one query/mutation on the editor.

Yesterday I was able to fix this issue, I would like to submit this pull request.

Thank you,
Sebastian.

**Comment by hwillson** [comment]
We love PR's @slorenzo - thanks!

**Reply by saeta-eth** [reply]
Great @hwillson. I already sent it, let me know your thoughts!
